gitgud/__main__.py

```diff
@@ -160,6 +162,7 @@ def handle_init(self, args):
         print('Welcome to Git Gud!')
         print()
 
+        init_tracking_json(self.file_operator)
         self.file_operator.get_level().setup(self.file_operator)
         show_tree()
```

Comment:
Remove these three lines and use `load_level` function here. `load_level` will needlessly write the level again, but that's a minor cost, and the change would improve readability.

Reply:
added init_tracking_json to load_level. Deleting these three lines.